lib/common/templates/markdown/table.md.template.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @typedef {import('../../../../types/DesignToken.ts').TransformedToken} TransformedToken
+ * @typedef {import('../../../../types/Config.ts').Config} Config
+ * @typedef {import('../../../../types/Config.ts').LocalOptions} LocalOptions
+ */
+
+/**
+ * @param {{
+ *   allTokens: TransformedToken[]
+ *   formatProperty: (token: TransformedToken) => string
```

Comment:
I don't think this is needed here, see other comment about not using `createPropertyFormatter`